packages/functions/extract/src/get-namespace-members.ts

```diff
@@ -0,0 +1,55 @@
+import type { Member } from "@acme/extract-schema";
+import type { ExtractFunction, Entities } from "./config";
+import type { Pagination, SourceControl } from "@acme/source-control";
+import type { LibSQLDatabase } from "drizzle-orm/libsql";
+import type { BetterSQLite3Database } from "drizzle-orm/better-sqlite3";
+
+export type GetMembersInput = {
+  namespaceName: string;
+  repositoryName: string;
+  repositoryId: number;
+  page?: number;
+  perPage?: number;
+};
+
+export type GetMembersOutput = {
+  members: Member[];
+  paginationInfo: Pagination;
+};
+
+export type GetMembersSourceControl = Pick<SourceControl, "fetchNamespaceMembers">;
+export type GetMembersEntities = Pick<Entities, "members" | "repositoriesToMembers">;
+
+export type GetMembersFunction = ExtractFunction<GetMembersInput, GetMembersOutput, GetMembersSourceControl, GetMembersEntities>;
```

Comment:
Do we want to rename this into GetNamespaceMembers to differentiate ? we could also expand the getMembers with a source parameter: type source = "namespace" | "repository"